app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
```

Comment:
Are these comments just to outline what paths exist? I'm not sure they are necessary. This seems to occur often, so I assume a command was run that generated the controllers, but any unused ones can be deleted, though they might be worth keeping.